bin/run-gem-minitests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Get names of modified files compared to the main branch.
+DIFF_DATA=$(git diff --name-status main)
```

Comment:
Should we convert this file to Ruby? I'm not the best at Bash, so this seems complicated to maintain.

I wonder if we could do something like this? (though we'd probably have to fiddle with regex)

```ruby
#!/usr/bin/env ruby

gems = `git diff --name-status main`.each_line.grep(/^bullet_train-/).uniq
gems.each do |gem|
  `cd #{gem} && bundle install && bin/rails test test/` # We're passing test/ to also run system tests.
end
```

Reply:
Done!

Unfortunately the eager loading issue is still a problem, so I think I'm just going to have to sit down with it and get as far as I can. There are a lot of moving parts so I'm not sure what needs to be included where, but if you have any insight on that side of things that would be super helpful! In the meantime, I'll try to look over it again and see what I can do.